doc/development/deprecations.rst

```diff
@@ -9,6 +9,41 @@ deprecations are listed below.
 Pending deprecations
 --------------------
 
+New operator arithmetic deprecations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This release completes the main phase of PennyLane's switchover to an updated approach for handling
+arithmetic operations between operators, check out the :ref:`Updated operators <new_opmath>` page
+for more details. The old system is still accessible via :func:`~.disable_new_opmath`. However, the
+old system will be removed in an upcoming release and should be treated as deprecated. The following
+functionality will explicitly raise a deprecation warning when used:
+
+* ``op.ops`` and ``op.coeffs`` will be deprecated in the future. Use ``op.terms()`` instead.
```

Comment:
Can we link to the `terms()` method in the docs?

Reply:
```suggestion
* ``op.ops`` and ``op.coeffs`` will be deprecated in the future. Use 
  :meth:`~.Operator.terms` instead.
```